ca/client.go

```diff
@@ -605,6 +608,34 @@ retry:
 	return &version, nil
 }
 
+// Capabilities performs the capabilities request to the CA and returns the
+// api.Capabilities struct.
+func (c *Client) Capabilities() (*api.CapabilitiesResponse, error) {
```

Comment:
Probably OK to return it as a value instead of a pointer.

Reply:
The way you recommend is the way we've done it for other types - like Version I think. So it makes sense to be consistent. But I hate that I'll have to remember to update >1 types. Makes it harder on developers in my opinion. If the general consensus is to go with a new type, then I'm happy to switch.

Reply:
Both Health and Version return a pointer so I think it's better to be consistent.